.github/workflows/docs.yml

```diff
@@ -0,0 +1,14 @@
+name: Build documentation
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  docs:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: build docs
+      run: TRAVIS_BRANCH=master ./tools/travis_update_docs.sh
```

Comment:
I fixed the script on master, it was pointing at the wrong url. It should just work now; of course it won't for you because it requires a token.